translations/nl-be.json

```diff
@@ -1354,6 +1354,18 @@
   "check-documents-to-be-sent-to-vp": "Vink de documenten aan die je naar het Vlaams Parlement wilt doorsturen.",
   "possible-pieces-missing": "Kijk na of de bovenstaande lijst volledig is.",
   "automatic-check-performed-possible-pieces-missing":"Op basis van een automatische controle lijken de volgende documenten te ontbreken. Wil je ze toch indienen? Laad ze dan op en stuur ze vervolgens door.",
+  "case-info": "Dossier info",
+  "design-of": "Ontwerp van",
+  "last-statusses": "Laatste statussen",
+  "to-flemish-parliament": "Naar Vlaams Parlement",
+  "number": "Nummer",
+  "accepted-in-plenary-meeting-on": "Aangenomen in plenaire vergadering op",
+  "ratified-and-announced-on": "Bekrachtigd en afgekondigd op",
+  "published-in-belgian-official-gazette": "Bekendgemaakt in het Belgisch Staatsblad op",
+  "updated-on-date": "Geüpdatet op {date}",
```

Comment:
We're using "Bijgewerkt op" throughout the application. I'd use that instead of "Geüpdatet op"